iree/vm/list_test.cc

```diff
@@ -386,6 +388,70 @@ TEST_F(VMListTest, ResizeVariant) {
 
 // TODO(benvanik): test ref get/set.
 
+// Tests pushing and popping ref objects.
+TEST_F(VMListTest, PushPopRef) {
```

Comment:
I guess this test is failing ASAN somehow?

https://source.cloud.google.com/results/invocations/7ee4c378-daec-48d3-a67e-52a3b4aa7ddb/targets/iree%2Fgcp_ubuntu%2Fcmake%2Flinux%2Fx86-swiftshader-asan%2Fpresubmit/log